index.js

```diff
@@ -19,8 +21,35 @@ mongoose
     // Before adding any recipes to the database, let's remove all existing ones
     return Recipe.deleteMany()
   })
-  .then(() => {
-    // Run your code here, after you have insured that the connection was made
+  .then(async(req,res) => {
```

Comment:
Note que estas usando async and away, el manejo de errores con estas promesas es diferente 
```suggestion
 //
 
  const myNewRecipe = {
      title: "Chocolate Chip Cookies",
      level: "Amateur Chef",
      ingredients: [
        "1/2 cup light brown sugar",
        "1 large egg",
        "2 tablespoons milk",
        "1 1/4 teaspoons vanilla extract",
        "2 cups semisweet chocolate chips"
      ],
      cuisine: "French",
      dishType: "dessert",
      image: "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fimages.media-allrecipes.com%2Fuserphotos%2F4398987.jpg&w=596&h=399.32000000000005&c=sc&poi=face&q=85",
      duration: 30,
      creator: "Chef Jennifer"
    }

const handleData= async ()=>{
 const recipeCreated = await Recipe.create(myNewRecipe);
    //console.log(recipeCreated.title);
    recipeCreated.forEach((element) => {
    console.log(element.title); //no puedo imprimir los titles aun, esto es por que nos devuelven un array y no un elemento por esto te recomiendo hacer un forEach o un for como aqui 😉 
  });


// Agregamos todas las recetas

const allRecipes = await Recipe.insertMany(data)
}
   

```